python/cuml/test/dask/utils.py

```diff
@@ -46,25 +46,29 @@ def to_cudf(df, r):
     return cudf.from_pandas(df)
 
 
-def dask_make_blobs(nrows, ncols, n_centers=8, cluster_std=1.0,
+def dask_make_blobs(nrows, ncols, n_centers=8, n_parts=None, cluster_std=1.0,
```

Comment:
Future change: move to dask.datasets or something like that?

Reply:
I'll create an issue for this. I had originally made this function to ease testing and it's different than what you might expect from sklearn's `make_blobs`. It would be nice to spend a little time making them more similar.